### PR TITLE
add resolution rules to market templates

### LINKED ISSUE #4188: add resolution rules to templates
suggested resolution rules to be added to resolution details

<https://www.figma.com/file/IpIAYVQPeQfuT5TTUboQpo4S/Market-creation?node-id=912%3A0>

image

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/4188

### CODE REVIEW

File: packages/augur-ui/src/modules/create-market/components/common.tsx [hunk 1]
```diff
@@ -992,3 +996,85 @@ export const CategoricalTemplate = (props: CategoricalTemplateProps) => {
...
+export interface ResolutionRulesProps {
+  newMarket: NewMarket;
+  onChange: Function;
+}
+
+export const ResolutionRules = (props: ResolutionRulesProps) => {
+  const { onChange, newMarket } = props;
+  const { template } = newMarket;
+  const { resolutionRules } = template;
+  if (
```
Comment (bthaile) [inline packages/augur-ui/src/modules/create-market/components/common.tsx]: not following the logic here.  are you seeing if there is anything in that resolutionRules?

if (Object.keys(resolutionRules).length === 0)

that would do it.

**Review by bthaile — COMMENTED** [review]
code looks good, will pull down and test.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). ✨

 This link will always point to this PR's latest changes: https://peek.run/7g6hv9w

**Review by bthaile — APPROVED** [review]
Lgtm

**Pull request merged**